### DISCUSSION TITLE
Cannot use SSR in _app.tsx

### DISCUSSION THREAD

**Discussion by thanhtutzaw** [vercel/next.js#54559] [Help]
### Summary

SSR can only work in pages . But why SSR in _app.tsx is not working ? 
I was trying to get token from cookies with nookies in SSR . I can get value in index.tsx . But not in app.tsx. 

![Screenshot (295)](https://github.com/vercel/next.js/assets/71011043/e64e03c8-8ca2-4e15-a2d3-7f12454acb4c)

**Comment by hyjoong** [comment]
According to the Next.js official documentation, _app.tsx does not support data fetching methods like getStaticProps or getServerSideProps.

https://nextjs.org/docs/pages/building-your-application/routing/custom-app#usage
![image](https://github.com/vercel/next.js/assets/70426440/3482cc01-a264-42ab-8161-9cf6ec34a30e)


Is there a specific reason why you need to use SSR in _app.tsx, or was it simply a test?
